packages/node/test/integration/uninstall-concurrent-mixed.spec.ts

```diff
@@ -0,0 +1,143 @@
+import { NetworkContextForTestSuite } from "@counterfactual/local-ganache-server";
+import { One } from "ethers/constants";
+import { parseEther } from "ethers/utils";
+
+import { Node } from "../../src";
+import { CONVENTION_FOR_ETH_TOKEN_ADDRESS } from "../../src/constants";
+import { InstallMessage, NODE_EVENTS, ProposeMessage } from "../../src/types";
+import { toBeLt } from "../machine/integration/bignumber-jest-matcher";
+
+import { setup, SetupContext } from "./setup";
+import {
+  collateralizeChannel,
+  constructUninstallRpc,
+  constructUninstallVirtualRpc,
+  createChannel,
+  installVirtualApp,
+  makeInstallCall,
+  makeProposeCall
+} from "./utils";
+
+expect.extend({ toBeLt });
+
+jest.setTimeout(15000);
+
+// TODO: this test isnt working bc of the concurrency issues that are being investigated now.
+// reenable this test when it works
+describe.skip("Concurrently uninstalling virtual and regular applications without issue", () => {
```

Comment:
do we want to merge skipped tests into master?

Reply:
I think it's okay as we intend for this to pass